vocode/streaming/streaming_conversation.py

```diff
@@ -1372,6 +1370,11 @@ def is_interrupt(self, transcription: Transcription):
             self.transcriber.get_transcriber_config().min_interrupt_confidence or 0
         )
 
+    def get_duration_spoken_seconds(self, time_started_speaking: float | None):
```

Comment:
let's make this a standalone function so readers can assume it doesn't rely on the millions of flags in this class

Reply:
I am going to just add a`@staticmethod` decorator.